core/build.gradle

```diff
@@ -251,6 +251,7 @@ task integrationTestJdbc(type: Test) {
     options {
         systemProperties(System.getProperties().findAll{it.key.toString().startsWith("scalardb")})
     }
+    maxHeapSize = "4g"
```

Comment:
I needed to increase the heap because of experiencing failure with the addition of the large BLOB integration tests for Oracle.